BoundingBox bounding_box
int32 red
int32 green
int32 blue